examples/plot_pulse_buffers.py

```diff
@@ -0,0 +1,96 @@
+from typing import Iterable
+
+import numpy as np
+from matplotlib import pyplot as plt
+from matplotlib import ticker
+from qat.purr.backends.echo import EchoEngine, get_default_echo_hardware
+from qat.purr.backends.live import LiveDeviceEngine
+from qat.purr.compiler.emitter import InstructionEmitter
+from qat.purr.compiler.runtime import get_builder
+from qat.purr.integrations.qasm import Qasm3Parser
+
+np.set_printoptions(
+    edgeitems=3000000, linewidth=100000000, formatter=dict(float=lambda x: "%.3g" % x)
+)
+
+import logging
+
+logging.disable()
```

Comment:
What's with this?

Reply:
oh yeah can remove, I just don't like most the logging when I'm testing, it's noisy